### ISSUE TITLE
Immobility Field drastically reduces decomposition

### ISSUE THREAD

**Issue by 89409** [studio-minus/ppg-bugs-and-requests#1209]
**Feature description**
Describe the requested feature. Is it a new spawnable item? Is it a new map? A new mechanic or system? How does it work?

When a body part gets covered in the field, it drastically reduces the decomposition time (pretty much a bandage but 5x more efficient).

**Feature purpose**
Describe the purpose here. What made you request this feature? What is its purpose? Are you sure it doesn't already exist in some form?

Allows for more creativity with builds (for example, a futuristic hospital with cryo chambers in it).

**Additional context**

**Comment by DindinYT37** [comment]
Yeah but then it wouldn't be just an immobility field. Not against it, just saying that the name wouldn't fit as well as before if this gets added.

**Comment by ppgplayer** [comment]
> Yeah but then it wouldn't be just an immobility field. Not against it, just saying that the name wouldn't fit as well as before if this gets added.

could be renamed to "Dimensional Field" or sum shit
